vite.config.ts

```diff
@@ -114,19 +115,92 @@ export default defineConfig(({ command: _buildOrServe }) => ({
 			},
 		},
 		{
-			name: 'update-animations-json',
-			handleHotUpdate({ file, server, read: _read }) {
-				if (file.endsWith('animations.json')) {
+			name: 'create-dist-files',
+			apply: 'serve',
+			buildStart() {
+				const files = [...moduleJSON.esmodules, moduleJSON.styles].flat()
```

Comment:
Can we be consistent here, use `...` on both or remove the spreader and just use `flat`